tools/msvs/msi/nodemsi/i18n/en-us.wxl

```diff
@@ -20,10 +20,14 @@
     <String Id="NodeRuntime_Description" Value="Install the core [ProductName] runtime (node.exe)."/>
 
     <String Id="npm_Title" Value="npm package manager"/>
-    <String Id="npm_Description" Value="Install npm, the recommended package manager for [ProductName]."/>
+    <String Id="npm_Description" Value="Install npm, a package manager for [ProductName]."/>
```

Comment:
This change does not relate to enabling Corepack by default (ie. we can recommend `npm` even though Corepack is being distributed). I'd suggest making a separate PR so we can discuss this change independently.

```suggestion
    <String Id="npm_Description" Value="Install npm, the recommended package manager for [ProductName]."/>
```